src/main/java/com/direwolf20/buildinggadgets/client/gui/ModeRadialMenu.java

```diff
@@ -40,26 +39,12 @@
 
 public class ModeRadialMenu extends GuiScreen {
 
-    private static final ResourceLocation[] signsBuilding = new ResourceLocation[]{
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/build_to_me.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/vertical_column.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/horizontal_column.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/vertical_wall.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/horizontal_wall.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/stairs.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/grid.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/surface.png")
-    };
-    private static final ResourceLocation[] signsExchanger = new ResourceLocation[]{
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/surface.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/vertical_column.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/horizontal_column.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/grid.png")
-    };
-    private static final ResourceLocation[] signsCopyPaste = new ResourceLocation[]{
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/copy.png"),
-        new ResourceLocation(BuildingGadgets.MODID, "textures/gui/mode/paste.png")
-    };
+    //TODO move to a enum of modes of Copy-Paste Gadget
```

Comment:
When?

Reply:
I will do it afterwards as I don't want to change the title of this PR :)